moveit_py/moveit/servo_client/teleop.py

```diff
@@ -52,12 +52,16 @@ class TeleopDevice(ABC, Node):
     This class expects both the `publish_command` and `record` methods to be overridden by inheriting classes.
     """
 
-    def __init__(self, node_name, device_name, device_config, ee_frame_name):
+    def __init__(self, node_name, device_name, device_config):
         super().__init__(node_name=node_name)
         self.device_name = device_name
         self.device_config = device_config
-        self.ee_frame_name = ee_frame_name
-
+        self.declare_parameters(
+            namespace=node_name, parameters=[("ee_frame_name", None)]
+        )
+        self.ee_frame_name = (
+            self.get_parameter("ee_frame_name").get_parameter_value().string_value
```

Comment:
This line might not be doing what you expect, here you are getting a parameter that has been set to None. Currently the `ee_frame_name` parameter from `servo.yaml` lives under the `servo_node`. 

The following [thread](https://answers.ros.org/question/340600/how-to-get-ros2-parameter-hosted-by-another-node/) suggests that you would need to create a service client to gain access to the parameters of another node via rclpy. This may have changed since. 

If the teleop device was started via a launch file, this wouldn't be an issue since we would include the node parameters there by default. The only drawback here is that we forgo starting devices from the Python library API.